__tests__/app.test.js

```diff
@@ -65,10 +68,21 @@ describe('GET /api/articles/:article_id', () => {
       expect(article).toHaveProperty('created_at')
       expect(article).toHaveProperty('votes')
       expect(article).toHaveProperty('article_img_url')
+
```

Comment:
Not bad at all! However, we can be more specific in this assertion, because we know what article we're getting back (article with ID 4), so we now what author, title etc. there should be